CMP/CmpWorkerService/App.config

```diff
@@ -1,148 +1,148 @@
+  <configSections>
+    <section name="entityFramework" type="System.Data.Entity.Internal.ConfigFile.EntityFrameworkSection, EntityFramework, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" requirePermission="false" />
+    
+  <!-- For more information on Entity Framework configuration, visit http://go.microsoft.com/fwlink/?LinkID=237468 --></configSections>
+  <connectionStrings>
+    <!-- Filled in by installer setup. A user gets created to access DB. You do not need to edit these connection strings 
+          unless you want to. Please follow the instructions in the Phoenix CMP WAP Extension Web.config file. -->
+    <add name="CMPContext" connectionString="" providerName="System.Data.SqlClient" />
+  </connectionStrings>
+  <appSettings>
+    <add key="CMPContextPassword" value="" />
+    <add key="MicrosoftMgmtSvcCmpContextPassword" value="" />
+    <add key="MicrosoftMgmtSvcStoreContextPassword" value="" />
+    <add key="KryptoCert" value=""/>
+
+    <!--This data is configurable to manipulate the way the AzureRefreshService class queries catalogue information from Azure-->
+    <add key="AzureCatalogueSyncTargetPublisher" value="MicrosoftWindowsServer" /> <!--Works like a search term. "Microsoft" will bring any publisher with "Microsoft" in its name-->
+    <add key="AzureCatalogueSyncTargetRegion" value="US" /> <!--Works like a search term. "West" will bring any region with "west" in its name-->
+    <add key="AzureCatalogueSyncTargetNumberOfPublishers" value="50" />
+
+    <!--Data below should be left with default values. For future use -->
+    <add key="AzureApi" value="ARM" />
+    <add key="PerformValidation" value="T" />
+    <add key="DisableSmartCardAuth" value="T" />
+    <add key="MovePagefile" value="T" />
+    <add key="MakeIpStatic" value="T" />
+    <add key="ProvisionDisks" value="T" />
+    <add key="AddUsersToGroups" value="T" />
+    <add key="ProcessSequences" value="T" />
+    <add key="VnetsMustBeActive" value="false" />
+    <add key="ProcessVmsActive" value="T" />
+    <add key="ClientSettingsProvider.ServiceUri" value="" />
+    <add key="CmpWorker.DomainJoin.DomainName" value="" />
+    <add key="CmpWorker.DomainJoin.UserName" value="" />
+    <add key="CmpWorker.DomainJoin.UserPassword" value="[KText]xxx[/KText]" />
+    <add key="CmpWorker.DomainJoin.OU" value="OU=aaa,OU=bbb,DC=ccc,DC=ddd,DC=eee,DC=fff" />
+    <add key="AUTOLOCALADMINUSERNAME" value="" />
+    <add key="AUTOLOCALADMINPASSWORD" value="" />
+    <add key="AUTOSUBNETNAME" value="TenantSubnet" />
+    <add key="DefaultStateTtlMinutes" value="120" />
+    <add key="DefaultVmRebootDwellTimeMinutes" value="3" />
+    <add key="DefaultCoreSafetyStockValue" value="10" />
+    <add key="BLOBSPERCONTAINERLIMIT" value="40" />
+    <add key="VMMTUSETTING" value="1438" />
+    <add key="VMSPERVNETLIMIT" value="180" />
+    <add key="EnforceAppAgAffinity" value="false" />
+    <add key="ImportAllAzureSubVms" value="false" />

```

Comment:
Also, is this a setting that we need to take care of during installation time? (i.e., does it affect the wizard?) If not and we only need to put this in the config, then no prob.